src/components/Auth/UsersCard/table/UsersTable.tsx

```diff
@@ -88,7 +88,14 @@ export const UsersTableRow: React.FC<
           onOpen={() => setMenuOpen(true)}
           onClose={() => setMenuOpen(false)}
           handle={
-            <IconButton theme="secondary" icon="more_vert" label="Open menu" />
+            <IconButton
+              theme="secondary"
+              icon="more_vert"
+              aria-label={`Open menu for user ${
+                user.displayName || user.email || user.phoneNumber
```

Comment:
Let's leave the phone number off - unless we prepend it with something that indicates it's a phone number.

Reply:
Gotcha; I'll add some additional text for when a user's Identifier is  a phone number